assets/scss/layout-builder.scss

```
@@ -602,6 +602,13 @@ figure.drupal-media.ck-widget {
   }
 }
 
+.entity-browser-modal.ui-dialog .ui-dialog-titlebar-close {
+  &:before,
+  &:after {
+    border-color: var(--jui-dialog-title-color);
```

We do have a practive to define a fallback color too, in case we do decouple variables in a single space
```suggestion
    border-color: var(--jui-dialog-title-color, #FFF);
```